sql/moz-fx-data-shared-prod/mozilla_vpn_derived/funnel_ga_to_subscriptions_v1/query.sql

```diff
@@ -70,27 +70,14 @@ subscriptions AS (
     normalized_campaign,
     normalized_content,
     IF(
-      COUNT(DISTINCT COALESCE(website_channel_group, "NULL")) = 1,
+      COUNT(DISTINCT website_channel_group) = 1,
       ANY_VALUE(website_channel_group),
       ERROR("website_channel_group must have one value per group")
     ) AS website_channel_group,
     COUNT(DISTINCT subscription_id) AS total_new_subscriptions,
     COUNT(
-      DISTINCT
-      CASE
-      WHEN
-        DATE(customer_start_date) < DATE(subscription_start_date)
-      THEN
-        subscription_id
-      ELSE
-        NULL
-      END
+      DISTINCT IF(DATE(customer_start_date) < DATE(subscription_start_date), subscription_id, NULL)
     ) AS returning_subscriptions,
-    IF(
-      COUNT(DISTINCT COALESCE(utm_content, "NULL")) = 1,
-      ANY_VALUE(utm_content),
-      ERROR("utm_content must have one value per group")
-    ) AS utm_content,
```

Comment:
Is it okay to remove a column like this?

Reply:
this column doesn't end up in the final table, so it's mostly just a waste of compute time